From b458c3798bc242d147b0fd940d668487c5806eea Mon Sep 17 00:00:00 2001
From: Lance Stringham <lstringham@nvidia.com>
Date: Wed, 15 Jan 2025 13:38:13 -0800
Subject: [PATCH] Update Winbond chip id string to match exact variant used by
 GB platform

Update the chip id string of the Winbond part to match the exact variant
used in our various GB platforms to ease BOM validation.
---
 drivers/mtd/spi/spi-nor-ids.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mtd/spi/spi-nor-ids.c b/drivers/mtd/spi/spi-nor-ids.c
index a3e903feb8..b62a31a37e 100644
--- a/drivers/mtd/spi/spi-nor-ids.c
+++ b/drivers/mtd/spi/spi-nor-ids.c
@@ -441,7 +441,7 @@ const struct flash_info spi_nor_ids[] = {
 	{ INFO("w25q512jv", 0xef4020, 0, 64 * 1024, 1024,
 			SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ |
 			SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) },
-	{ INFO("w25q512jvfm", 0xef7020, 0, 64 * 1024, 1024,
+	{ INFO("w25q512jvbm", 0xef7020, 0, 64 * 1024, 1024,
 			SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ |
 			SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) },
 	{ INFO("w25q01jv", 0xef4021, 0, 64 * 1024, 2048,
@@ -481,7 +481,7 @@ const struct flash_info spi_nor_ids[] = {
 			SECT_4K | SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) },
 	{ INFO("w25q512jv", 0xef4020, 0, 64 * 1024, 1024,
 			SECT_4K | SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) },
-	{ INFO("w25q512jvfm", 0xef7020, 0, 64 * 1024, 1024,
+	{ INFO("w25q512jvbm", 0xef7020, 0, 64 * 1024, 1024,
 			SECT_4K | SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) },
 	{ INFO("w25q01jv", 0xef4021, 0, 64 * 1024, 2048,
 			SECT_4K | SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) },
-- 
2.34.1

